Rakefile

```diff
@@ -56,6 +57,44 @@ def format_links(text)
   text.gsub(/{([^}]+)}/, '[`\1`](#\1)')
 end
 
+desc "Generate markdown docs for Bolt's command line options"
+task :reference_docs do
+  parser = Bolt::BoltOptionParser.new({})
+  @commands = {}
+
+  Bolt::CLI::COMMANDS.each do |subcommand, actions|
+    actions << nil if actions.empty?
+
+    actions.each do |action|
+      command = [subcommand, action].compact.join(' ')
+      help_text = parser.get_help_text(subcommand, action)
+
+      options = help_text[:flags].map do |option|
+        switch = parser.top.long[option]
+
+        {
+          short: switch.short.first,
+          long: switch.long.first,
+          arg: switch.arg,
+          desc: switch.desc.map { |d| d.gsub("<", "&lt;") }.join("<br>")
+        }
+      end
+
+      @commands[command] = {
+        banner: help_text[:banner].gsub("<", "&lt;").gsub("\n", "<br>"),
```

Comment:
Not sure why we need to sub the newlines? Maybe just sub the last `\n` with `<br>` i think it is causing only a single line from the banner to show up.

Reply:
That was to make sure when there is a single newline between lines that it displays correctly in the markdown doc. I'm not sure how that translates to the static site, so I'll remove the sub and do a preview of the site to make sure formatting isn't wonky.